compiler/pipes/code-gen.cpp

```diff
@@ -154,34 +132,12 @@ void CodeGenF::prepare_generate_function(FunctionPtr func) {
     ? func->file_id->owner_lib->headers_dir() + func->header_name
     : func->subdir + "/" + func->header_name;
 
-  my_unique(&func->static_var_ids);
-  my_unique(&func->global_var_ids);
-  my_unique(&func->local_var_ids);
+  std::sort(func->static_var_ids.begin(), func->static_var_ids.end());
```

Comment:
Why did you remove unique?

Reply:
Because these vars are already unique. I mean, that kphp_assert(count_before == count_after) is true always.